src/modules/esl-media/core/esl-media.ts

```diff
@@ -374,8 +374,9 @@ export class ESLMedia extends ESLBaseElement {
     (this._provider) && this._provider.safeSeekTo(time);
   }
 
+  // TODO: simplify - ESLMediaQuery is lazy for primitives, it make sense to init it unconditionally
```

Comment:
ESLMediaQuery 2.0.0 has lightweight constant queries and ootb cache. So it's better to have 'ALL' or empty string default for query with assumption that the query is always available instead of managing everything manually.